Extension/src/LanguageServer/ui.ts

```diff
@@ -36,10 +36,18 @@ interface ConfigurationStatus {
 }
 
 export class UI {
-    private configStatusBarItem: vscode.StatusBarItem;
-    private browseEngineStatusBarItem: vscode.StatusBarItem;
-    private intelliSenseStatusBarItem: vscode.StatusBarItem;
-    private referencesStatusBarItem: vscode.StatusBarItem;
+    private configStatusBarItem!: vscode.LanguageStatusItem;
```

Comment:
What does `!:` mean?

Reply:
It basically means that variable can get away with not being initialized in the constructor

Reply:
@Bernardin-MS I still don't understand. Do you happen to know where that's documented online or what to search for?

Reply:
[Non-null assertion operator](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#non-null-assertion-operator)

Reply:
The non-null assertion operator is only used in expressions, but I found the similar feature at https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#definite-assignment-assertions .